architecture/README.md

```diff
@@ -31,8 +31,11 @@ To achieve this, the View should be bound using the `KalugaViewModelLifecycleObs
 For convenience default implementations for `Activity`, `Fragment`, and `DialogFragment` exist (`KalugaViewModelActivity`, `KalugaViewModelFragment`, and `KalugaViewModelDialogFragment` respectively).
 Alternatively the user can call `didResume()` and `didPause()` on the ViewModel manually in the `onResume()` and `onPause()` methods.
 
-On iOS automatic setup was omitted due to ObjC classes requiring to be final.
-To use it, the user must manually call the ViewModels `didResume`/`didPause` in `viewDidAppear`/`viewDidDisappear`.
+On iOS automatic setup can be achieved by binding an `UIViewController` to the viewModel using `addLifecycleManager`.
```

Comment:
I think it's good to have consistency with cases in `viewModel`, `ViewModel` and `viewmodel` (also view controller)

Reply:
added case consistency :)